azure-pipelines.yml

```diff
@@ -10,7 +10,7 @@ variables:
   # If updating VERILATOR_VERSION, OPENOCD_VERSION, TOOLCHAIN_VERSION or RUST_VERSION
   # update the definitions in util/container/Dockerfile as well.
   #
-  VERILATOR_VERSION: 4.104
+  VERILATOR_VERSION: 4.210
```

Comment:
does it make sense to update the documentation also to recommend this version?

Reply:
The documentation pulls the version number from `tool_requirements.py`, which is now updated as well.